### PR TITLE
Commit current state of exp example apps EXP-3667

### PR DESCRIPTION
**Pull request by brianfeister**

### CODE REVIEW

**Comment by adamgallowayscala** [comment]
@brianfeister socialmedia-app needs to be removed and the bower_components of weather-app needs to be removed

File: README.md [hunk 1]
```diff
@@ -1,13 +1,57 @@
...
+
+
+# UUIDs
+
+The following is a list of UUIDs for each global app in EXP. These apps are available publically without credentials.
 
 | App Template | UUID |
 |--------------|------|
 |audio|3415bd0b-f567-4f53-896a-2344b24645be|
-|dayplan|4d0d80e8-fdba-46a2-8a54-90fdab96390d
```
Comment (adamgallowayscala) [inline README.md]: why remove this and schedule?

**Review by adamgallowayscala — CHANGES_REQUESTED** [review]
also needs slideshow changed to playlist in the readme table of uuids

File: README.md (hunk 1 above)
Comment (brianfeister) [inline README.md]: To be clear, I copy-pasted the README.md, and this was the state of current develop

Comment (adamgallowayscala) [inline README.md]: ok, this should be merged from production

Comment (adamgallowayscala) [inline README.md]: still probably broken there but we don't want app code from develop

Comment (brianfeister) [inline README.md]: gotcha, doing that now

**Comment by brianfeister** [comment]
@adamgallowayscala Changes I made:

1. Copy-pasted the state of production (this had no impact, they are in sync)
1. Removed `bower_components` from weather app
1. Removed `socialmedia-app`
1. Updated README `slideshow` => `playlist`

**Review by adamgallowayscala — APPROVED** [review]

**Comment by adamgallowayscala** [comment]
👍

**Pull request merged**